includes/core/payments/details.php

```diff
@@ -69,7 +71,7 @@ public function set_tags() {
 
 			'total-amount' => array(
 				'type'  => array( 'all' ),
-				'value' => isset( $this->payment->charge->amount ) ? simpay_formatted_amount( $this->payment->charge->amount, $this->payment->get_currency() ) : '',
+				'value' => isset( $this->payment->charge->amount ) ? simpay_format_currency( simpay_convert_amount_to_dollars( $this->payment->charge->amount ), $this->payment->get_currency() ) : '',
```

Comment:
This should be double checked.

Reply:
Looks correct. Part of amount conversion refactoring done in early 2018 in Pro.

https://github.com/wpsimplepay/WP-Simple-Pay-Pro-3/commit/73be1bc9aa1a60ef7401ddad4bdd43fcc2ebd4c7#diff-c456e49aee98642a1d8fac8d13b9b0ca
https://github.com/wpsimplepay/WP-Simple-Pay-Pro-3/commit/ec9c0fed173c1b078b96c6995d0416868e0fff95#diff-c456e49aee98642a1d8fac8d13b9b0ca